apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -239,7 +239,29 @@ export class SpecialAnnouncementActionBlock extends Component {
       </div>
     );
 
-    return (
+    const hasIncompleteApplication = !!sessionStorage['Teacher1920Application'];
+
+    return hasIncompleteApplication ? (
+      <TwoColumnActionBlock
+        imageUrl={pegasus(
+          '/shared/images/fill-540x289/teacher-announcement/professional-learning-2019-3.jpg'
+        )}
+        subHeading={
+          'Finish your application to the Professional Learning Program'
+        }
+        subHeadingSmallFont={true}
+        description={
+          'We noticed you started your application to the Code.org Professional Learning Program.\
+          Finish your application while seats last!'
+        }
+        buttons={[
+          {
+            url: studio('/pd/application/teacher'),
```

Comment:
Since we are already on dashboard, we can just use the URL without the `studio()` part.